db/seeds/api.controller.js

```diff
@@ -1,8 +1,22 @@
 const endpointsJson = require("../../endpoints.json");
+const { selectTopics } = require("./api.model");
 
-function getApi(req, res) {
+console.log("in controller");
+
+exports.getApi = (req, res) => {
   console.log(endpointsJson, "<--- endpoints json");
   res.status(200).send({ endpoint: endpointsJson });
-}
+};
 
-module.exports = getApi;
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      if (topics.length === 0 || !topics) {
```

Comment:
This isn't how to handle errors. If you get to this part, this has resolved. Refer back to the error handling notes.